js/main.js

```diff
@@ -40,6 +48,37 @@ Site = {
 
 };
 
+Site.Shop = {
+  init: function() {
+    var _this = this;
+
+    this.Search.init();
+  },
+
+  Search: {
+    init: function() {
+      var _this = this;
+
+      if ($('#searchform').length) {
+        _this.bindFocus();
+      }
+    },
+
+    bindFocus: function() {
```

Comment:
for me it looks like this
![screen shot 2016-08-12 at 4 10 53 pm](https://cloud.githubusercontent.com/assets/7727722/17639465/69a037d8-60a7-11e6-852f-5c2d1eafda8f.png)